### PR TITLE
Adding instruction for zsh for bazel build mixer

### PR DESCRIPTION
**Pull request by linsun**
As titled :)

### CODE REVIEW

File: devel/README.md [hunk 1]
```diff
@@ -232,7 +232,7 @@ cd $(ISTIO)/mixer
 bazel build ...
 ```
 
-This figures out what it needs to do and does not need any input from you.
+If you are using zsh, use ```bazel build //...``` or  ```bazel build "..."```. This build command figures out what it needs to do and does not need any input from you.
```
Comment (ldemailly) [inline devel/README.md]: single backquote is probably enough btw

Comment (linsun) [inline devel/README.md]: actually, single backquote doesn't work for me (zsh).

**Comment by linsun** [comment]
@ldemailly I'll make the change you suggested.  I had thought of that too when making the patch. :)

File: devel/README.md [hunk 2]
```diff
@@ -229,10 +229,10 @@ Using Mixer as an example:
 
 ```shell
 cd $(ISTIO)/mixer
-bazel build ...
+bazel build //...
 ```
 
-This figures out what it needs to do and does not need any input from you.
+This build command figures out what it needs to do and does not need any input from you.
```
Comment (lizan) [inline devel/README.md]: nit: trailing space

Comment (linsun) [inline devel/README.md]: hmm, it doesn't work for me, here is my zsh version:

→ zsh --version
zsh 5.2 (x86_64-apple-darwin16.0)

Perhaps you are on a newer version?    Good catch on the nit. :)

File: devel/README.md (hunk 1 above)
Comment (ldemailly) [inline devel/README.md]: I meant for .md formatting as a command but it's irrelevant now

File: devel/README.md (hunk 2 above)
Comment (lizan) [inline devel/README.md]: `zsh 5.2 (x86_64-apple-darwin16.0)` same for me.
Probably you have the `rationalise-dot` config in your `zshrc`?

**Pull request merged**